integration/gethnetwork/geth_network_test.go

```diff
@@ -95,32 +95,37 @@ func TestGethTransactionIsMintedOverRPC(t *testing.T) {
 	}
 
 	// wallet should be prefunded
-	w := datagenerator.RandomWallet()
+	w := datagenerator.RandomWallet(1337)
```

Comment:
shouldn't we use the constant everywhere we use the "1337" port

Reply:
The `integration` package has the constants for the simulation. 
The `gethnetwork` package does not have a constant for the chainID, it's acutally hardcoded in the genesis file.

But I added a constant to the test file 👍